.github/workflows/ci.yml

```diff
@@ -152,7 +148,8 @@ jobs:
       fail-fast: false
       matrix:
         # Note: these versions must match scripts/packaging/engine-versions.txt
-        unreal: ['4.27', '5.0', '5.1', '5.2', '5.3', '5.4', '5.5']
+        # Note: Currently 5.0 is not supported due to ue4-docker issue https://github.com/adamrehn/ue4-docker/issues/373
```

Comment:
It's old enough we could prob remove support btw. Looking at numbers seems to be safe.

Or just call it 'best effort'